### PR TITLE
installation skip gitignore overwrite

### PR DESCRIPTION
**Pull request by sensorario**

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -28,7 +44,6 @@ git clone https://github.com/AlessandroYorba/Despacio.git && cd despacio.git/col
 
 cp -f $PWD/gitignore_global ~/.gitignore_global;
```
Comment (nephilem90) [inline install.sh:45]: rewrite always? move this line after `. ./update-gitconfig.sh`

File: install.sh [hunk 2]
```diff
@@ -5,14 +5,30 @@ clear
...
-read committer_email
-sed "s/{{committer_user}}/$committer_user/g" <./gitconfig.dist >./temp
-sed "s/{{committer_email}}/$committer_email/g" <./temp >./gitconfig
-
+if [ $# -eq 0 ]
+  then
+    echo "Install starts ..."
+  else
+    echo $1
+    if [ $1 == "--gitconfig" ]
```
Comment (nephilem90) [inline install.sh:13]: if .gitconfig not exist you can always run `. ./update-gitconfig.sh`

update with this

`if [ $1 == "--gitconfig" ] || [ ! -f ~/.gitconfig ]`

File: install.sh (hunk 1 above)
Comment (sensorario) [reply install.sh:45]: Could you please open a PR for this?

File: install.sh (hunk 2 above)
Comment (sensorario) [reply install.sh:13]: Could you please open a PR for this?

**Pull request merged**